pkger/clone_resource.go

```diff
@@ -832,11 +832,14 @@ func convertCellView(cell influxdb.Cell) chart {
 		ch.Kind = chartKindMosaic
 		ch.Queries = convertQueries(p.Queries)
 		ch.Colors = stringsToColors(p.ViewColors)
+		ch.HoverDimension = p.HoverDimension
 		ch.XCol = p.XColumn
 		ch.GenerateXAxisTicks = p.GenerateXAxisTicks
 		ch.XTotalTicks = p.XTotalTicks
 		ch.XTickStart = p.XTickStart
 		ch.XTickStep = p.XTickStep
+		ch.YLabelColumnSeparator = p.YLabelColumnSeparator
+		ch.YLabelColumns = p.YLabelColumns
```

Comment:
Same question, should we do anything special for empty?

Reply:
No. Empty array will be interpreted correctly by the UI.